### ISSUE TITLE
Stickers on exhibit pages should go to item record

### ISSUE THREAD

**Issue by EWB** [EWB/peopleshistoryarchive#15]

**Comment by EWB** [comment]
done... and also switched to display only one image per node (useful in the event that an item record has more than one image)

**Comment by CatherineTedford** [comment]
It's looking amazing!  Today is the first time I tried to access the PHA on my ipad, and it's really smooth!

Thanks, Eric.

Cathy

Catherine Tedford, Director
Richard F. Brush Art Gallery
St. Lawrence University
23 Romoda Drive
Canton, NY 13617
www.stlawu.edu/gallery<http://www.stlawu.edu/gallery>

On Aug 29, 2015, at 12:58 PM, EWB <notifications@github.com<mailto:notifications@github.com>> wrote:


done... and also switched to display only one image per node (useful in the event that an item record has more than one image)

**Comment by EWB** [comment]
Nice. Good to hear.

 Note that I'm spending most of the day tweaking different aspects – so some things may look funky for a bit – while they're still in the process of being configured.

E



On Aug 29, 2015, at 1:38 PM, CatherineTedford <notifications@github.com<mailto:notifications@github.com>> wrote:

It's looking amazing! Today is the first time I tried to access the PHA on my ipad, and it's really smooth!

Thanks, Eric.

Cathy

Catherine Tedford, Director
Richard F. Brush Art Gallery
St. Lawrence University
23 Romoda Drive
Canton, NY 13617
www.stlawu.edu/gallery<http://www.stlawu.edu/gallery><http://www.stlawu.edu/gallery>

On Aug 29, 2015, at 12:58 PM, EWB <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com>> wrote:


done... and also switched to display only one image per node (useful in the event that an item record has more than one image)

**Comment by CatherineTedford** [comment]
Sorry you have to work on Saturday.  But I do appreciate your doing this!  Let me know if there is anything I can do to help!

Cathy

Catherine Tedford, Director
Richard F. Brush Art Gallery
St. Lawrence University
23 Romoda Drive
Canton, NY 13617
www.stlawu.edu/gallery<http://www.stlawu.edu/gallery>

On Aug 29, 2015, at 1:42 PM, EWB <notifications@github.com<mailto:notifications@github.com>> wrote:

Nice. Good to hear.

Note that I'm spending most of the day tweaking different aspects – so some things may look funky for a bit – while they're still in the process of being configured.

E



On Aug 29, 2015, at 1:38 PM, CatherineTedford <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com>> wrote:

It's looking amazing! Today is the first time I tried to access the PHA on my ipad, and it's really smooth!

Thanks, Eric.

Cathy

Catherine Tedford, Director
Richard F. Brush Art Gallery
St. Lawrence University
23 Romoda Drive
Canton, NY 13617
www.stlawu.edu/gallery<http://www.stlawu.edu/gallery><http://www.stlawu.edu/gallery><http://www.stlawu.edu/gallery>

On Aug 29, 2015, at 12:58 PM, EWB <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com><mailto:notifications@github.com>> wrote:


done... and also switched to display only one image per node (useful in the event that an item record has more than one image)

**Comment by EWB** [comment]
No worries – I save the cool projects for the weekends :-).



On Aug 29, 2015, at 1:52 PM, CatherineTedford <notifications@github.com<mailto:notifications@github.com>> wrote:

Sorry you have to work on Saturday. But I do appreciate your doing this! Let me know if there is anything I can do to help!

Cathy

Catherine Tedford, Director
Richard F. Brush Art Gallery
St. Lawrence University
23 Romoda Drive
Canton, NY 13617
www.stlawu.edu/gallery<http://www.stlawu.edu/gallery><http://www.stlawu.edu/gallery>

On Aug 29, 2015, at 1:42 PM, EWB <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com>> wrote:

Nice. Good to hear.

Note that I'm spending most of the day tweaking different aspects – so some things may look funky for a bit – while they're still in the process of being configured.

E



On Aug 29, 2015, at 1:38 PM, CatherineTedford <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com><mailto:notifications@github.com>> wrote:

It's looking amazing! Today is the first time I tried to access the PHA on my ipad, and it's really smooth!

Thanks, Eric.

Cathy

Catherine Tedford, Director
Richard F. Brush Art Gallery
St. Lawrence University
23 Romoda Drive
Canton, NY 13617
www.stlawu.edu/gallery<http://www.stlawu.edu/gallery><http://www.stlawu.edu/gallery><http://www.stlawu.edu/gallery><http://www.stlawu.edu/gallery>

On Aug 29, 2015, at 12:58 PM, EWB <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com><mailto:notifications@github.com><mailto:notifications@github.com>> wrote:


done... and also switched to display only one image per node (useful in the event that an item record has more than one image)

**Comment by CatherineTedford** [comment]
Yeah, same.  Heading over to ODY soon to work on a paper....  It never ends!  But in a good way.  😊

Catherine Tedford, Director
Richard F. Brush Art Gallery
St. Lawrence University
23 Romoda Drive
Canton, NY 13617
www.stlawu.edu/gallery<http://www.stlawu.edu/gallery>

On Aug 29, 2015, at 1:54 PM, EWB <notifications@github.com<mailto:notifications@github.com>> wrote:

No worries – I save the cool projects for the weekends :-).



On Aug 29, 2015, at 1:52 PM, CatherineTedford <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com>> wrote:

Sorry you have to work on Saturday. But I do appreciate your doing this! Let me know if there is anything I can do to help!

Cathy

Catherine Tedford, Director
Richard F. Brush Art Gallery
St. Lawrence University
23 Romoda Drive
Canton, NY 13617
www.stlawu.edu/gallery<http://www.stlawu.edu/gallery><http://www.stlawu.edu/gallery><http://www.stlawu.edu/gallery>

On Aug 29, 2015, at 1:42 PM, EWB <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com><mailto:notifications@github.com>> wrote:

Nice. Good to hear.

Note that I'm spending most of the day tweaking different aspects – so some things may look funky for a bit – while they're still in the process of being configured.

E



On Aug 29, 2015, at 1:38 PM, CatherineTedford <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com><mailto:notifications@github.com><mailto:notifications@github.com>> wrote:

It's looking amazing! Today is the first time I tried to access the PHA on my ipad, and it's really smooth!

Thanks, Eric.

Cathy

Catherine Tedford, Director
Richard F. Brush Art Gallery
St. Lawrence University
23 Romoda Drive
Canton, NY 13617
www.stlawu.edu/gallery<http://www.stlawu.edu/gallery><http://www.stlawu.edu/gallery><http://www.stlawu.edu/gallery><http://www.stlawu.edu/gallery><http://www.stlawu.edu/gallery>

On Aug 29, 2015, at 12:58 PM, EWB <notifications@github.com<mailto:notifications@github.com><mailto:notifications@github.com><mailto:notifications@github.com><mailto:notifications@github.com><mailto:notifications@github.com>> wrote:


done... and also switched to display only one image per node (useful in the event that an item record has more than one image)
